docs/components/movement-sensor/_index.md

```diff
@@ -497,7 +497,7 @@ properties, err := myMovementSensor.Properties(context.Background(), nil)
 
 ### GetAccuracy
 
-Get the accuracy of the sensor (and/or precision, depending on the sensor model).
+Get the precision and reliability metrics of the movement sensor, including various parameters to assess the sensor's accuracy in different dimensions.
```

Comment:
This wording sounds kinda like it conflates precision and accuracy. Would this suggestion be correct? Or is there a reason to emphasize precision over accuracy despite the method name?
```suggestion
Get the reliability metrics of the movement sensor, including various parameters to assess the sensor's accuracy and precision in different dimensions.
```